src/main/java/com/example/hbs/model/Hotel.java

```diff
@@ -19,26 +19,26 @@
     private List<Booking> bookings;
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(nullable = false, referencedColumnName = "Id", name = "user_id")
+    @JoinColumn(name = "user_id", nullable = false, referencedColumnName = "Id")
```

Comment:
i think JPA automatically maps id column by default, no need for referencedColumnName here, check once